### PR TITLE
TELCODOCS-961 - Configuring PGT CRs with compliance duration settings

### PR DESCRIPTION
**Pull request by aireilly**
https://issues.redhat.com/browse/TELCODOCS-961

Merge to main, CP to enterprise-4.11+

Preview (temporary TOC location, will be updated in https://github.com/openshift/openshift-docs/pull/49805): http://file.emea.redhat.com/aireilly/td-961/scalability_and_performance/ztp-deploying-disconnected.html#ztp-configuring-pgt-compliance-eval-timeouts_ztp-deploying-disconnected

### CODE REVIEW

File: modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_content-type: PROCEDURE
+[id="ztp-configuring-pgt-compliance-eval-timeouts_{context}"]
+= Configuring compliance evaluation timeouts for PolicyGenTemplate CRs
+
+You can configure PolicyGenTemplate CRs with duration settings for how long a cluster can be in a state of policy compliance or non-compliance before {rh-rhacm} re-evaluates the applied cluster policies. The default value for the `noncompliant` state is 10 seconds. The default value for the `compliant` state is 10 minutes. To disable the evaluation interval, set the value to `never`.
```
Comment (tliu2021) [inline modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc:9]: The duration settings are applied to a ConfigurationPolicy and they are not cluster wide configuration.

Comment (aireilly) [reply modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc]: Updated. PTAL

**Comment by mcornea** [comment]
lgtm

File: modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a Git repository where you manage your custom site configuration data.
+
+.Procedure
+
+. To configure the evaluation interval for all policies in a `PolicyGenTemplate` CR, add `evaluationInterval` to the `spec` field, setting appropriate `compliant` and `noncompliant` values, for example:
```
Comment (mjpytlak) [inline modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc:25]:
```suggestion
. To configure the evaluation interval for all policies in a `PolicyGenTemplate` CR, add `evaluationInterval` to the `spec` field, and then set the appropriate `compliant` and `noncompliant` values. For example:
```

File: modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+   - fileName: SriovSubscription.yaml
+     policyName: "sriov-sub-policy"
+     evaluationInterval:
+       compliant: never
+       noncompliant: 10s
+----
+
+. Commit the `PolicyGenTemplate` CRs files in the Git repository and push your changes.
+
+.Verification
```
Comment (mjpytlak) [inline modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc:50]: I believe your preview might be out of date. The Verification steps are not in the published output. Please update, so I can verify that it looks OK.

**Review by mjpytlak — COMMENTED** [review]
@aireilly This is well done. One copy edit (purely a suggestion). Please just update your preview. I suspect that you added the Verification steps after you pushed it.

File: modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc (hunk 3 above)
Comment (mjpytlak) [reply modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc:50]: /remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Comment by bergerhoffer** [comment]
@aireilly @lpettyjo This PR was marked for 4.11 only, but it went into main. If something goes into main, it has to also go into 4.12 too (otherwise the update should have gone directly into enteprise-4.11).

Can we CP this PR to enterprise-4.12 too?

**Comment by aireilly** [comment]
> @aireilly @lpettyjo This PR was marked for 4.11 only, but it went into main. If something goes into main, it has to also go into 4.12 too (otherwise the update should have gone directly into enteprise-4.11).
> 
> 
> 
> Can we CP this PR to enterprise-4.12 too?

Good catch!! Thanks

**Comment by bergerhoffer** [comment]
Thanks @aireilly, I merged it!

**Pull request merged**